Automattic/jetpack
# Add/my jetpack for 2yr plans

Fixes 1201210512993648-as-1204690966572292

## Proposed changes:

* Add Jetpack Security T1 and Jetpack Backup T1 2-year plans to missing variables so the My Plan page displays the right plan.

<img width="400" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/d35cc925-bd1a-436a-8763-1aa3739d55a1">

<img width="400" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/3c51ff4a-b621-46a4-9799-53ae6dd7121a">

### Other information:


## Jetpack product discussion


N/a

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Checkout this branch locally
* Run `jetpack build plugins/jetpack && jetpack install plugins/jetpack`
* Run `jetpack watch plugins/jetpack`
* Open `projects/plugins/jetpack/_inc/client/my-plan/index.js`
* Add `sitePlan = 'jetpack_security_t1_bi_yearly;` below line 26
* Visit `/wp-admin/admin.php?page=jetpack#/my-plan` and make sure it displays the right data for the sitePlan. You can also change the `sitePlan` to `'jetpack_backup_t1_bi_yearly'` and confirm it displays the My Plan for VaultPress Backup
